modules/registry-configuring-storage-vsphere.adoc

```diff
@@ -42,13 +47,12 @@ $ oc get pod -n openshift-image-registry
 +
 [NOTE]
 =====
-If the storage type is `emptyDIR`, the replica number cannot be greater than `1`.
-If the storage type is `NFS`, and you want to scale up the registry Pod by setting
-`replica>1` you must enable the `no_wdelay` mount option. For example:
-
+* If the storage type is `emptyDIR`, the replica number cannot be greater than `1`.
+* If the storage type is `NFS`, you must enable the `no_wdelay` and `root_squash` mount options. For example:
++
```

Comment:
@huffmanca and @qinpingli - I made a small change here (removing ref in NFS for `replica>1`) based on Alex's BZ comment: https://bugzilla.redhat.com/show_bug.cgi?id=1850510#c15

PTAL and let me know if you both are good with this revision. Thanks.

Reply:
LGTM

Reply:
This looks good to me!